helmfile.d/40-realtime-dashboard.yaml

```diff
@@ -0,0 +1,40 @@
+bases:
+- ../environments.yaml
+
+---
+repositories:
+  - name: radar
+    url: https://radar-base.github.io/radar-helm-charts
+  - name: cp-radar
+    url: https://radar-base.github.io/cp-helm-charts
+
+helmDefaults:
+  atomic: false
+  timeout: 240
+  kubeContext: {{ .Values.kubeContext }}
+
+releases:
+  - name: radar-jdbc-connector-agg
+    chart: radar/radar-jdbc-connector
+    version: {{ .Values.radar_jdbc_connector_agg._chart_version }}
+    installed: {{ .Values.radar_jdbc_connector_agg._install }}
+    values:
+      - "../etc/radar-jdbc-connector-agg/values.yaml"
+      - {{ .Values.radar_jdbc_connector_agg | toYaml | indent 8 | trim }}
+    set:
+      - name: kafka_num_brokers
+        value: {{ .Values.kafka_num_brokers }}
+      - name: timescaledb.password
```

Comment:
should this be updated in line with the other jdbc connector (`jdbc.password`) ?

Reply:
Good spot, fixed